lms/djangoapps/bulk_email/tasks.py

```diff
@@ -195,6 +195,12 @@ def perform_delegate_email_batches(entry_id, course_id, task_input, action_name)
     if total_recipients <= settings.BULK_EMAIL_JOB_SIZE_THRESHOLD:
         routing_key = settings.BULK_EMAIL_ROUTING_KEY_SMALL_JOBS
 
+    # Weird things happen if we allow empty querysets as input to emailing subtasks
```

Comment:
You explained what kind of "weird things" happen in a comment on the PR.  Can you include a clearer explanation in the code as well?  It would be nice if a future developer could use the comment to determine whether this code is still needed.